src/main/scala/com/codacy/client/stash/client/StashClient.scala

```diff
@@ -149,6 +149,9 @@ class StashClient(apiUrl: String, authenticator: Option[Authenticator] = None) {
     try {
       val baseRequest = Http(url).method(method).params(params)
 
+      if (acceptAllCertificates)
+        baseRequest.option(HttpOptions.allowUnsafeSSL)
```

Comment:
What does the option method returns? Does the .option does a "set"? Is this enough?

Reply:
No, it just modifies the existing object